content/docs/03-cluster-profiles/10-byoos/10-image-builder.md

```diff
@@ -0,0 +1,316 @@
+    |---|----|----|
+    | `osImageOverride` | The ID of the image to use as the base OS layer. This is the ID of the image as assigned in the infrastructure environment it belongs to. Example: `ami-0f4804aff4cf9c5a2`. | string|
+    | `osName` | The name of the OS distribution. Example: `rhel`. | string |
+    | `osVersion`| The version of the OS distribution. Example: `8` | string|
+
+    <br />
+
+    ```yaml
+    pack:
+      osImageOverride: "ami-0f4804aff4cf9c5a2"
+      osName: "rhel"
+      osVersion: "8"
+    ```
+
+
+  ![View of the cluster profile wizard](/clusters_byoos_image-builder_cluster-profile-byoos-yaml.png)
+
+
+17. Click on **Next layer** to add the Kubernetes layer.
+
+
+18. Select the desired Kubernetes distribution and version. Click on the **</\>** button to reveal the YAML editor.
+
+
+19. If you use a commercial OS, such as RHEL, you must activate the subscription. You can use the `preKubeadmCommands` section of the Kubernetes layer to issue commands before Kubernetes is started. In this example, the [RHEL Subscription Manager CLI](https://access.redhat.com/solutions/253273) is used to register the compute instance. Other OS may have different activation methods. Review the respective OS's documentation to learn more about activation methods.
+
+  <br />
+
+
+  ```yaml
+  preKubeadmCommands:
+   -  subscription-manager register --username=REPLACE_ME --password=REPLACE_ME
+  ```
+
+  <br />
+
+
+  ![A view of the Kubernetes layer's YAML with a custom preKubeadmCommand](/cluster-profiles_byoos_image-builder_kubernetes-layer-example.png)
+
+  <br />
+
+17. Complete the remainder of the cluster profile creation wizard by selecting the next cluster profile layers.
+
+You now have a cluster profile that uses the custom image you created using the [Kubernetes Image Builder](https://image-builder.sigs.k8s.io/introduction.html) project. 
+
+<br />
+
+<WarningBox>
+
+When deploying a host cluster, choosing the appropriate cloud provider and region where the image was distributed is critical to successfully launching a cluster using a custom image in the cluster profile. Failure to do so may result in Palette's inability to launch a cluster.
+
+</WarningBox>
+
+# Validation
+
+Use the following steps to validate your custom image.
+
+1. You can validate that the custom image is working correctly by deploying a compute instance in the respective infrastructure provider you created the image in using the custom image. Review the compute instance logs to learn more about the problems if you encounter any issues.
+
+
+2. Next, deploy a host cluster that uses the cluster profile you created containing the custom image. Verify the cluster is deployed correctly and without any issues. If you encounter any problems, review the event logs of the cluster to gain more details about the issue. Check out the [Deploy a Cluster](/clusters/public-cloud/deploy-k8s-cluster/) tutorial for additional guidance on deploying a host cluster.
```

Comment:
```suggestion
2. Next, deploy a host cluster that uses the cluster profile you created containing the custom image. Verify that the cluster is deployed correctly and without any issues. If you encounter any problems, review the event logs of the cluster to gain more details about the issue. Check out the [Deploy a Cluster](/clusters/public-cloud/deploy-k8s-cluster/) tutorial for additional guidance on deploying a host cluster.
```

Reply:
I think this "that" is unwarranted in this case.